manifests/base/config/metrics_allowlist.yaml

```diff
@@ -58,6 +58,8 @@ data:
       - etcd_server_proposals_committed_total
       - etcd_server_proposals_applied_total
       - etcd_server_quota_backend_bytes
+      - grpc_server_started_total
+      - grpc_server_handled_total
```

Comment:
grpc_server_handled_total has large number of time series data
in my 3M3W cluster, it returns 2431